include/gdsb/experiment.h

```diff
@@ -68,6 +68,22 @@ template <typename It> void out(std::string name, It begin, It end, std::ostream
     stream << "]" << std::endl;
 }
 
+//! Use this function to output ranges in yaml format.
+template <typename It, typename F>
+void out(
+    std::string name, It begin, It end, std::ostream& stream = std::cout, F&& access = [](auto const& e) { return e; })
+{
+    stream << name << ": [";
+
+    if (begin != end)
+    {
+        std::for_each(begin, end - 1, [&](auto const& e) { stream << access(e) << ", "; });
```

Comment:
Can you write a test for this function? The test for `out()` using iterators can be found here: https://github.com/hu-macsy/graph-ds-benchmark/blob/main/test/experiment_tests.cpp#L31

Reply:
There's now a test similar to the other tests.